app/pathogen/arbovirus/analyze/recharts/change-in-median-seroprevalence-over-time-graph.tsx

```diff
@@ -115,16 +115,18 @@ export const ChangeInMedianSeroprevalenceOverTimeGraph = (): React.ReactNode =>
       ([arbovirus, dataPointsForAParticularArbovirus]) => [
         arbovirus,
         groupDataPointsIntoTimeBuckets({
-          dataPoints: dataPointsForAParticularArbovirus.map((dataPoint) => ({
-            ...dataPoint,
-            groupingTimeInterval: {
-              intervalStartDate: parseISO(dataPoint.sampleStartDate),
-              intervalEndDate: parseISO(dataPoint.sampleEndDate),
+          dataPoints: dataPointsForAParticularArbovirus
+            .filter((dataPoint) => dataPoint.sampleStartDate && dataPoint.sampleEndDate)
```

Comment:
This was the problem :/

If you're wondering why Chrome was affected and Firefox was not, I think it's just some browser jank and I don't want to read into the behaviour of the different functions when you pass in undefined just to find out why this happened, I don't think that's a super good use of time.

I could make some kind of rule for when the start / end date isn't present but I actually read the study in question and there is a sampling end date in the text, we just haven't recorded one. Also the only reasonable solution here would be to set one to the other and I don't like that we have inconsistent strategies everywhere and throwing magic in different places for dealing with this problem.

My plan here instead is to read through as much as I can and try to get sampling start and end dates on everything and bring them to data. After that, I'll set up the ETL to reject anything with a missing sampling start date and end date and maybe generate some kind of report that can be sent to the data folks. Anyways, this is all dependent on how data feels but just dropping my feelings here.

Reply:
Ok I agree! So far devs did not really touch the data. There is a field in the airtable that makrs a record as ready for extraction or not based on what data is required vs optional. I like the approach of report and not allowing null fields BUT we may need to do that if the Data team feels like we should

@harrietware any input?